### PR TITLE
executor: Add unit test case for ShowExecutor.

### PR DESCRIPTION
**Pull request by shenli**
Add a mocked session manager for unit test.
PR #4739 could also get benefit from this PR.

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: executor/executor_pkg_test.go [hunk 1]
```diff
@@ -61,3 +68,81 @@ func (s *testExecSuite) TestMergeHandles(c *C) {
 		c.Assert(kr.EndKey, DeepEquals, ekr.EndKey)
 	}
 }
+
+// mockSessionManager is a mocked session manager which is used for test.
```
Comment (Darren) [inline executor/executor_pkg_test.go]: Can this mocked sessionManager be moved into util/mock package so that it can be resued?

Comment (shenli) [reply executor/executor_pkg_test.go]: But this is only useful in this package. Is there any other package need to use it?

Comment (shenli) [reply executor/executor_pkg_test.go]: We could move it out to mock package when we meet the scenario.

**Comment by shenli** [comment]
@tiancaiamao PTAL

**Comment by tiancaiamao** [comment]
LGTM

File: util/mock/context.go [hunk 2]
```diff
@@ -153,7 +154,12 @@ func (c *Context) GetStore() kv.Storage {
 
 // GetSessionManager implements the context.Context interface.
 func (c *Context) GetSessionManager() util.SessionManager {
-	return nil
+	return c.sm
+}
+
+// SetSessionManager set the session manager.
```
Comment (zimulala) [inline util/mock/context.go]: s/set/sets

**Pull request merged**